run/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+if [ -z "$*" ]; then
+  echo "Expecting one or more arguments"
+  exit 1
+fi
+# shellcheck source=scripts/spack_setup.sh
+source "$GITHUB_WORKSPACE"/scripts/spack_setup.sh fetch
+spack install "$SPACK_SPEC"
```

Comment:
we can remove this and the following two lines, as we'll leave this as a generic function